code/modules/spells/spell.dm

```diff
@@ -65,6 +65,8 @@
 	var/invocation
 	/// What is shown in chat when the user casts the spell, only matters for INVOCATION_EMOTE
 	var/invocation_self_message
+	/// if true, doesn't garble the invocation sometimes with backticks
+	var/garbled_invocation = TRUE
```

Comment:
why not something like garble_chance and then you can just use prob(garble_chance), set it to 0 where you dont want it

Reply:
i like that